site/content/3.13/data-science/arangographml/ui.md

```diff
@@ -0,0 +1,268 @@
+
+1.  **Featurization**: Your raw graph data is transformed into numerical representations that the model can understand.
+    *   The system iterates over your selected vertices and converts their attributes: booleans become `0` or `1`, numbers are normalized, and text attributes are converted into numerical vectors using sentence transformers.
+    *   All of these numerical features are then combined (concatenated).
+    *   Finally, **Incremental PCA** (Incremental Principal Component Analysis a dimensionality reduction technique) is used to reduce the size of the combined features, which helps remove noise and keep only the most important information.
+
+2.  **Training**: The model learns from the graph's structure by sampling and aggregating information from each node's local neighborhood.
+    *   For each node, GraphSAGE looks at connections up to **2 hops away**.
+    *   Specifically, it uniformly samples up to **25 direct neighbors** (depth 1) and for each of those, it samples up to **10 of their neighbors** (depth 2).
+    *   By aggregating feature information from this sampled neighborhood, the model creates a rich "embedding" for each node that captures both its own features and its role in the graph.
+
+## Limitations
+
+*   **Edge Attributes**: The current version of ArangoGraphML does not support the use of edge attributes as features.
+*   **Dangling Edges**: Edges that point to non-existent vertices ("dangling edges") are not caught during the featurization analysis. They may cause errors later, during the Training phase.
+*   **Prediction Scope**: Predictions can only be run in batches on the graph. It is not possible to request a prediction for a single document on-the-fly (Example, via an AQL query).
+*   **Memory Usage**: Both featurization and training can be memory-intensive. Out-of-memory errors can occur on large graphs with insufficient system resources.
+
+## The GraphML Workflow
+
+The entire process is organized into sequential steps within a **Project**, giving you a clear path from data to prediction:
+
+1.  **Featurization:** Select your data and convert it into numerical features.
+2.  **Training:** Train a GraphSAGE model on the features and graph structure.
+3.  **Model Selection:** Evaluate the trained models and choose the best one.
+4.  **Prediction:** Use the selected model to generate predictions on your data.
+5.  **Scheduling (Optional):** Automate the prediction process to run at regular intervals.
+
+## Creating a GraphML Project
+
+To create a new GraphML project using the ArangoDB Web Interface, follow these steps:
+
+1. From the left-hand sidebar, select the database where you want to create the project.
+2. In the left-hand navigation menu, click **Data Science Suite** to open the GraphML project management interface, then click Run GraphML.
```

Comment:
The term "Data Science Suite" needs to be replaced with "GenAI Suite".
As "Run GraphML" is a UI label, it should be written in bold.